product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -0,0 +1,59 @@
+| Organization | Payment method added                                                                             | Organization owner/admin         |
+| Organization | Personal access key is expiring                                                                  | Organization owner               |
+| Organization | Machine user access key is expiring                                                              | Organization owner               |
+| Project      | Upcoming maintenance upgrade on a cluster (24hr)                                                 | Project owner/editor             |
+| Project      | Successful maintenance upgrade on a cluster                                                      | Project owner/editor             |
+| Project      | Failed maintenance upgrade on a cluster                                                          | Project owner/editor             |
+| Project      | Paused cluster will automatically reactivated in 24 hours                                        | Project owner/editor             |
+| Project      | Paused cluster was automatically reactivated                                                     | Project owner/editor             |
+| Project      | You must set up the encryption key permission for your CMK-enabled cluster                       | Project owner/editor             |
+| Project      | Key error with CMK-enabled cluster                                                               | Project owner and project editor |
+| Project      | User is invited to a project (displays only to the Project owner)                                | Project owner                    |
+| Project      | New role is assigned to you                                                                      | Project user                     |
+| Project      | Role is unassigned from you                                                                      | Project user                     |
+| Project      | Failed connection to third-party monitoring integration (and future non-monitoring integrations) | Project owner/editor             |
+
+## Configuring notifications
+
+The project owners/editors and organization owners/admins can configure the notifications for the events visible to them. They can choose if they want to receive notifications in the UI or email or both. They can also configure email notifications for their teams with in their organization. 
+
+Organization/Project owners can add one or more team email addresses to their organizations/projects. They can enable/disable different notification types to be sent to the team email address(es) they provide.
+
+Project level notifications are to be configured for a project.
+
+Notification settings made by a user is applicable only to that user. If an email notification is enabled, the email is send to the email address associated with their login. 
```

Comment:
```suggestion
Notification settings made by a user are applicable only to that user. If an email notification is enabled, emails will be sent to the email address associated with their login. 
```

Reply:
As per the docs style guide, we don't use future tense.

Reply:
Ok, got it. thanks